pkgs/development/libraries/adns/default.nix

```diff
@@ -21,6 +21,10 @@ stdenv.mkDerivation {
   # http://thread.gmane.org/gmane.linux.distributions.nixos/1328 for details.
   doCheck = false;
 
+  postInstall = if stdenv.isDarwin then ''
```

Comment:
A simpler form of this construct is `stdenv.lib.optionalString stdenv.isDarwin ''....''`. Also, I don't believe that `postInstall` is supposed to be set to `null`. Rather, I'd use the empty string to disable it (which is what `optionalString` does).